content/3_Silver/Func_Graphs.mdx

```diff
@@ -720,6 +720,74 @@ It's also possible to use `floyd(x)` to generate answers for all vertices in the
 containing `x` without using adjacency lists.
 
 <LanguageSection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+vector<int> res;
+vector<int> arr;
+
+void fillUp(int curr, int firstCycle) {
```

Comment:
1. add comments from the java version
2. use snake_case

Reply:
Done.